junit-platform-console/src/main/java/org/junit/platform/console/options/TestConsoleOutputOptionsMixin.java

```diff
@@ -51,11 +51,25 @@ static class ConsoleOutputOptions {
 		@Option(names = "-details-theme", hidden = true)
 		private Theme theme2 = DEFAULT_THEME;
 
+		@Option(names = "--redirect-stdout", paramLabel = "FILE", description = "Redirect tests stdout to a file.")
+		private Path stdout;
+
+		@Option(names = "-redirect-stdout", hidden = true)
+		private Path stdout2;
+
```

Comment:
We're only maintaining this kind of options for backwards compatibility (we switched to option parsers at some point) but we shouldn't introduce them for new options.

```suggestion
```